src/effects/VST/VSTEffect.cpp

```diff
@@ -1037,7 +1039,14 @@ return GuardedCall<bool>([&]{
 });
 }
 
-size_t VSTEffect::ProcessBlock(EffectSettings &,
+
+bool VSTEffectInstance::ProcessFinalize() noexcept
+{
+   return VSTInstanceBase::DoProcessFinalize();
```

Comment:
I think there is no longer any need to make `DoProcessFinalize` a separate function.

Reply:
Removing it at this stage is not so simple, because DoProcessFinalize is in VSTInstanceBase and it is used by VSTEffect; but VSTEffect does not know VSTEffectInstance... we'd need to wait until step 5